xarray/plot/utils.py

```diff
@@ -160,7 +160,7 @@ def _determine_cmap_params(plot_data, vmin=None, vmax=None, cmap=None,
     """
     import matplotlib as mpl
 
-    calc_data = np.ravel(plot_data[~pd.isnull(plot_data)])
+    calc_data = np.ravel(plot_data[np.isfinite(plot_data)])
```

Comment:
I think [`pd.notnull`](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.notnull.html#pandas.notnull) should work here?

Reply:
`pd.notnull(np.inf)` is True but we want it to be False. `calc_data` needs to contain all data values that are not inf/nan and `np.isfinite` does that.